assembly/compliance-tool/src/main/java/org/eclipse/sw360/antenna/frontend/compliancetool/sw360/updater/ClearingReportGenerator.java

```diff
@@ -22,7 +22,8 @@
     private static final String CLEARING_DOC_SUFFIX = "_clearing.json";
 
     Path createClearingDocument(SW360Release release, Path targetDir) {
-        Path clearingDocument = targetDir.resolve(release.getName() + release.getVersion() + CLEARING_DOC_SUFFIX);
+        String clearingDocumentName = release.getName() + "_" + release.getVersion() + CLEARING_DOC_SUFFIX;
+        Path clearingDocument = targetDir.resolve(clearingDocumentName.replace("/", "_"));
```

Comment:
I just tested this PR on a Windows machine and should the releases name contain a "\" backslash, the same problem occurs. This case should also be handled